app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -1233,10 +1243,12 @@ class StatePlayerRecyclerViewAssembler private constructor(
      * answer.
      */
     fun addCelebrationForCorrectAnswers(
+      accessibilityChecker: AccessibilityChecker,
```

Comment:
I suggest injecting this in the Builder's Factory and passing it along. That way it doesn't need to be null, and it simplifies using this assembler.

Although it means the checker will still be enabled for cases when the celebration isn't enabled, this should be fine since the runtime cost is very low & we'll still not actually use it.

Reply:
Done, moved it.